test/test_transforms_v2_consistency.py

```diff
@@ -83,35 +82,6 @@ def __init__(
         supports_pil=False,
         make_images_kwargs=dict(DEFAULT_MAKE_IMAGES_KWARGS, dtypes=[torch.float]),
     ),
-    ConsistencyConfig(
```

Comment:
All these configs are now obsolete. Going forwards, whenever we port a new test to `test_*_refactored.py`, we can also remove the configs here.